theano/scan_module/scan_opt.py

```diff
@@ -351,14 +378,14 @@ def process_node(self, fgraph, node):
                 remove=[node],
                 reason='scanOp_pushout_nonseqs_ops')
             return True
-        elif to_keep == []:
+        elif not to_keep:
```

Comment:
@caglar is right, it is a recommendation of PEP8.
In that case, you could actually use `to_keep_set` here instead.